app/Components/ChatApp.js

```diff
@@ -42,7 +42,7 @@ class ChatApp extends React.Component {
 
 ChatApp.propTypes = {
   socket: React.PropTypes.object,
-  user: React.PropTypes.object
+  user: React.PropTypes.string
```

Comment:
If these are required, you can append `.isRequired` also, which will then raise an error in development if you forget to pass them.  The linter can also detect those kinds of mistakes as well.